framework-cloud/cloud-deployment-plugin/pom.xml

```diff
@@ -38,5 +38,10 @@
       <artifactId>maven-plugin-annotations</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
It is problem in this case because this plugin will be often used externally. So, I think it would be better to add all dependencies to its pom and allow user to select cloud environment using parameter in Maven pom.